### ISSUE TITLE
Resolve form accessibility violations

### ISSUE THREAD

**Issue by lawhorkl** [uclibs/scholar_uc#1590]
### Descriptive summary

Our recent accessibility scan of Scholar 3 reveals 105 total accessibility issues, 2 of these are violations in our forms and not having a label attribute.

We need to resolve them before we push to prod.

A good tool while working on this is https://achecker.ca/checker/index.php to allow you to make sure your changes solve the issue.

The following is a list of all form accessibility issues.
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044761
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044762
